prov/verbs/src/verbs_eq.c

```diff
@@ -520,6 +520,10 @@ fi_ibv_eq_cm_process_event(struct fi_ibv_eq *eq,
 			return fi_ibv_eq_xrc_connected_event(eq, cma_event,
 							     entry, len, acked);
 		entry->info = NULL;
+		if (priv_datalen) {
+			cm_hdr = priv_data;
+			memcpy(&entry->data, cm_hdr->data, cm_hdr->size);
+		}
```

Comment:
`fi_ibv_eq_copy_event_data` copies the data already. Why do we need this?

Reply:
Note that fabtests has a cm_data test that verifies the CM connect/accept data exchange.

Reply:
Thanks a lot for your answer. Indeed, it seems that the code already implements CM accept data exchange in `fi_ibv_eq_copy_event_data`. I will do further testing.
For now, I'm closing the PR. 
Sorry for the noise and thanks again for your review.